Userland/Applications/SoundPlayer/PlaylistWidget.h

```diff
@@ -31,7 +31,6 @@ class PlaylistModel : public GUI::Model {
     Vector<M3UEntry> m_playlist_items;
 
     static DeprecatedString format_filesize(u64 size_in_bytes);
```

Comment:
:eyes:  `AK::human_readable_size` :eyes:

Reply:
(not asking you to make that change ofc, just caught my eye)

Reply:
_The yakbait, it never ends! :scream:_